content/influxdb/v2.4/get-started/write.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Get started writing data
+seotitle: Write data | Get started with InfluxDB
+list_title: Write data
+description: >
+  Get started writing data to InfluxDB by learning about line protocol and using
+  tools like the InfluxDB UI, `influx` CLI, and InfluxDB API.
+menu:
+  influxdb_2_4:
+    name: Write data
+    parent: Get started
+    identifier: get-started-write-data
+weight: 101
+metadata: [2 / 5]
+related:
+  - /influxdb/v2.4/write-data/
+  - /influxdb/v2.4/write-data/best-practices/
+  - /influxdb/v2.4/reference/syntax/line-protocol/
+  - /{{< latest "telegraf" >}}/
+---
+
+InfluxDB provides many different options for ingesting or writing data, including
+the following:
+
+- Influx user interface (UI)
+- InfluxDB HTTP API
+- `influx` CLI
```

Comment:
did you want to add links to point people to CLI documentation?  same with UI and API??